src/ConfigCatClientOptions.ts

```diff
@@ -2,11 +2,17 @@ import { ConfigCatConsoleLogger } from "./ConfigCatLogger";
 import { IConfigCatLogger, IAutoPollOptions, ILazyLoadingOptions, IManualPollOptions, LogLevel } from "./index";
 import COMMON_VERSION from "./Version";
 
+export enum DataGovernance {
```

Comment:
please export or move this type into index.

Reply:
done